test/helper/setup.bash

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# -------------------------------------------------------------------
+
+function __initialize_variables() {
+  function __check_if_set() {
+    if [[ ${!1+set} != 'set' ]]
+    then
+      echo "ERROR: (helper/setup.sh) '${1:?No variable name given to __check_if_set}' is not set" >&2
+      exit 1
+    fi
+  }
+
+  __check_if_set 'REPOSITORY_ROOT'
+  __check_if_set 'IMAGE_NAME'
+  __check_if_set 'CONTAINER_NAME'
+
+  TEST_TIMEOUT_IN_SECONDS=${TEST_TIMEOUT_IN_SECONDS:-120}
+  NUMBER_OF_LOG_LINES=${NUMBER_OF_LOG_LINES:-10}
+  SETUP_FILE_MARKER="${BATS_TMPDIR:?}/$(basename "${BATS_TEST_FILENAME:?}").setup_file"
+}
+
+__initialize_variables
```

Comment:
Please note: this is a completely new directory (`test/helper/` and not `test/test_helper`). I want to keep the old helper until all tests are migrated.

I want a file that just takes care of container initialization. I do not really want this to be mixed up with other helpers as it makes maintenance easy.

Reply:
Yes, I'm aware of the intention and support that. As well as the transitional stage where the original `test_helper` is to remain (_ideally not for an extended period of time to avoid maintenance concerns_).

With the new changes you've made this only leaves `TEST_TIMEOUT_IN_SECONDS` and `NUMBER_OF_LOG_LINES` as vars covered in `common.bash`, not as big of a concern for me though, so this is fine now :+1:

Reply:
I must have missed them. I can move them to `setup.bash` too :)